src/data/finset/basic.lean

```diff
@@ -394,6 +394,11 @@ protected theorem induction_on {α : Type*} {p : finset α → Prop} [decidable_
   (s : finset α) (h₁ : p ∅) (h₂ : ∀ ⦃a : α⦄ {s : finset α}, a ∉ s → p s → p (insert a s)) : p s :=
 finset.induction h₁ h₂ s
 
+theorem induction_on' {α : Type*} {p : finset α → Prop} [decidable_eq α]
```

Comment:
Presumably this wants `@[elab_as_eliminator]`?

Reply:
I guess if the others have it, then might as well put it on this one